package.json

```diff
@@ -31,9 +31,11 @@
     "glob-parent": ">=6.0.2",
     "nth-check": ">=2.0.1",
     "nuxt": "^2.15.8",
-    "prismjs": "^1.26.0"
+    "prismjs": "^1.26.0",
+    "test": "^0.6.0"
```

Comment:
question: Do we need this dependency?

Reply:
Removed... not sure where that came from.